core/Makefile

```diff
@@ -151,6 +151,12 @@ protobuf: $(PROTO_SRCS)
 
 modules: protobuf $(MODULE_OBJS) $(MODULE_LIBS)
 
+GIT_VERSION := \#define VERSION \
+	\"$(shell git describe --abbrev=4 --dirty --always --tags)\"
```

Comment:
Can you add a guard against cases where Bess is not a git repo to make sure the build does not fail, and perhaps a default version is used. It is possible to download tarballs even from github, and sometimes this is required in some environments.